pkgs/development/compilers/cudatoolkit/redist/find_outputs/__main__.py

```diff
@@ -0,0 +1,4 @@
+from find_outputs.__main__ import main
+
+if __name__ == "__main__":
+    main()
```

Comment:
why is this file necessary?

Reply:
That was poor packaging on my part -- I wanted to be able to do `python3 ./find_outputs` to run the project, which required having a `__main__.py` in the root directory. Much better now that it's packaged as a flake: https://github.com/ConnorBaker/cuda-redist-find-features